Exoft.Gamification.Api.Common/Models/ReturnPageModel.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Exoft.Gamification.Api.Common.Models
+{
+    public class ReturnPageModel<T>
+    {
```

Comment:
Lets think from client side... what we need there... 
* get list of paged items 
* know what's the current page number
* know that there are next pages or not. 
Given current implementation we covers 1st and 2nd but not 3rd case
Please update model with needed data